packages/design-system/src/components/DateField/DateField.tsx

```diff
@@ -1,19 +1,18 @@
-import * as React from 'react';
+import { FormControl, FormControlPropKeys } from '../FormControl/FormControl';
+import defaultDateFormatter from './defaultDateFormatter';
```

Comment:
I just realized that Eslint doesn't automatically check the Typescript codes like it does for the React codes during the `git commit` stage. However, github actions does automatically does the eslint check here. This would be be fixed by moving the `import defaultDateFormatter from './defaultDateFormatter';` below the `import React from 'react';` line